libsql-ffi/bundled/bindings/bindgen.rs

```diff
@@ -3325,6 +3325,17 @@ pub struct libsql_wal_methods {
             zBuf: *mut ::std::os::raw::c_uchar,
             pnLog: *mut ::std::os::raw::c_int,
             pnCkpt: *mut ::std::os::raw::c_int,
+            pCbData: *mut ::std::os::raw::c_void,
```

Comment:
nitpick: right above, the busy handler is first passed as a function pointer, and the next parameter is its data. It would be more consistent to use the same convention for xCb and pCbData